helm/fdb-operator/templates/manager/manager.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: system
```

Comment:
Is it the best practice to create a namespace when deploying things through Helm, or is this an artifact of the defaults from kubebuilder?

Reply:
If we talk specifically this namespace creation its not required.